make-release.sh

```diff
@@ -8,16 +8,10 @@ ORGANIZATION="eclipse"
 
  # KEEP RIGHT ORDER!!!
 DOCKER_FILES_LOCATIONS=(
-    che-server/dockerfiles/endpoint-watcher
-    che-server/dockerfiles/keycloak
-    che-server/dockerfiles/postgres
     che-server/dockerfiles/che
 )
 
 IMAGES_LIST=(
```

Comment:
since this array and the one above it is now only a single value, we could probably simplify this down to 1 variable for `QUAY_IMAGE=quay.io/eclipse/che-server`  and remove `DOCKER_FILES_LOCATIONS` as an array for a for loop.

Reply:
done